### PR TITLE
Add Nomis boost interaction

### PR DESCRIPTION
**Pull request by unchase**

### CODE REVIEW

**Comment by unchase** [comment]
Any updates?

File: projects/apps/Nomis.py [hunk 1]
```diff
@@ -13,6 +13,10 @@
     metrics=[
         SmartContractInteraction(
             "Interaction", "EQBykWvdmoyFD2kB4BJbdrxRqyWKzSLDn-SgwNvznRfbsaKv"
+        ), 
+        SmartContractInteraction( 
+            "Nomis Boost", 
+            "UQBPIFKq_HLbqoksBqPGIMjFjTco2NH7o0ckV5Iocj_A22J2"
```
Comment (aleksandrkaekhtin) [inline projects/apps/Nomis.py:19]: please use bounceable format for the address

Comment (unchase) [reply projects/apps/Nomis.py]: Done

**Comment by unchase** [comment]
@aleksandrkaekhtin Done.
Could you please merge the PR?

**Pull request merged**